mscore/debugger/debugger.cpp

```diff
@@ -51,11 +52,11 @@
 #include "libmscore/rest.h"
 #include "libmscore/score.h"
 #include "libmscore/segment.h"
-#include "libmscore/slur.h"
 #include "libmscore/sig.h"
-#include "libmscore/spacer.h"
+#include "libmscore/slur.h"
 #include "libmscore/stem.h"
 #include "libmscore/stemslash.h"
+#include "libmscore/spacer.h"
```

Comment:
Why is 'spacer.h' after 'stemslash.h'? Shouldn't this be alphabetical?

Reply:
It should indeed. Might have been a rebase issue or just dyslexia ;-)